x-pack/legacy/plugins/uptime/server/lib/helper/get_histogram_interval.ts

```diff
@@ -7,13 +7,26 @@
 import DateMath from '@elastic/datemath';
 import { QUERY } from '../../../common/constants';
 
+export const parseRelativeEndDate = (_dateEnd: string) => {
```

Comment:
Why the leading `_` in `_dateEnd`? Why not `dateEnd`? I assume it's so we can use `const dateEnd` later. I think it would be cleaner to come up with more descriptive variable names. Maybe make the argument `dateEndStr` for instance.

Reply:
done